home/templates/home/_recent_incidents_section.html

```diff
@@ -6,7 +6,7 @@ <h2 class="homepage-section-header">{{ label|default:"Recent Incidents" }}</h2>
 	<ul class="article-carousel__items">
 		{% for incident in incidents %}
 			<li class="article-carousel__item article-carousel__item--latest">
-				{% include "incident/_incident_card.html" with size="collapsible" details="small" incident=incident photo=incident.teaser_image only %}
+				{% include "incident/_incident_card.html" with size="collapsible" details="small" incident=incident photo=incident.teaser_graphic.0 only %}
```

Comment:
Did we change _incident_ teaser images into graphics (with chart support) as well? I'm not seeing that update to `IncidentPage` elsewhere in this code.